tests/settings.spec.ts

```diff
@@ -1,12 +1,28 @@
 import { expect, test } from "@playwright/test";
+import { checkPageLoaded, loadInitialPageFromBrowser } from "./shared.ts";
 
-test("changed settings are preserved across refresh", async ({ page }) => {
+test("changed settings are preserved across refresh", async ({ browser }) => {
```

Comment:
This test wasn't passing, according to Alex it was unreliable on `main` too. So I've made some incidental changes to make it more reliable I think. Certainly the changes to shared I think are valuable, maybe it's overkill to be ready for either avoid doubling back or not defaulting to true

Reply:
(Worth adding a comment in this test about why we have to be prepared for either initial checkbox state?)